code/modules/hydroponics/grown/plum.dm

```diff
@@ -0,0 +1,26 @@
+/obj/item/seeds/plum
+	name = "pack of plum seeds"
+	desc = "These seeds grow into plum trees."
+	icon_state = "seed-plum"
+	species = "plum"
+	plantname = "Plum Tree"
+	product = /obj/item/food/grown/plum
+	lifespan = 55
+	endurance = 35
+	growthstages = 6
```

Comment:
```suggestion

```

you can delete this, this is the default value of the parent

Reply:
Plum has no parent, it's a new file?

Reply:
Plum's parent is `/obj/item/seeds`, wherever that's defined. The values will come from there.

Reply:
Oooh. Okay! Gotcha. I looked at the grown file, did not think to look at seeds. Makes sense. Thanks for pointing that out.